src/topology/subset_properties.lean

```diff
@@ -393,6 +393,36 @@ by rw union_eq_Union; exact compact_Union (λ b, by cases b; assumption)
 lemma is_compact.insert (hs : is_compact s) (a) : is_compact (insert a s) :=
 compact_singleton.union hs
 
+/-- If `V : ι → set α` is a decreasing family of closed compact sets then any neighborhood of
+`⋂ i, V i` contains some `V i`. -/
+lemma exists_subset_nhd_of_compact {ι : Type*} [nonempty ι] {V : ι → set α} (hV : directed (⊇) V)
+  (hV_cpct : ∀ i, is_compact (V i)) (hV_closed : ∀ i, is_closed (V i))
```

Comment:
It always feels strange to me to have assumptions saying both that something is compact, and that it is closed, because in my usual world the former implies the latter. Can you specify in the docstring that you use this formulation instead of `[t2_space]` because you need it for non-separated situations?

Reply:
Yes, I almost wrote this comment yesterday but was too lazy. Actually I don't even need it in non-separated situations, I only followed the context in that file. Do you think I should restate it with "normal" assumptions in another file where `t2_space` is defined?

Reply:
Yes, I think it would be a good idea to restate it in `separation.lean` (and add pointers in both directions in the docstrings).